webapp/channels/src/packages/mattermost-redux/src/client/rudder.ts

```diff
@@ -45,13 +37,14 @@ export class RudderTelemetryHandler implements TelemetryHandler {
         rudderAnalytics.track('event', properties, options);
     }
 
-    trackFeatureEvent(userId: string, userRoles: string, featureName: string, event: string, props?: any) {
+    trackFeatureEvent(userId: string, userRoles: string, featureName: string, event: string, props: Record<string, unknown> = {}) {
```

Comment:
I like this type better, should we change the TrackEvent one as well?

Reply:
changed that as well